service/src/main/kotlin/io/provenance/explorer/service/PulseMetricService.kt

```diff
@@ -656,12 +655,20 @@ class PulseMetricService(
             range = range,
             atDateTime = atDateTime
         ) {
-            // TODO no great way to get participants by date/block
-            // TODO refactor to "active" accounts - i.e. doing stuff on the network
+            val days = 30L
+            val startDate = nowUTC().minusDays(days).toLocalDate()
+
+            val rangeSpan = rangeSpanFromCache(startDate, PulseCacheType.PULSE_PARTICIPANTS_METRIC, MetricRangeType.MONTH, days)
```

Comment:
Could you take a look at refactoring the `rangeSpanFromCache` function - the `range` param is only used in an error messages and seems unnecessary.